### DISCUSSION TITLE
AUR package

### DISCUSSION THREAD

**Discussion by an-prata** [GuardKenzie/miniplayer#41] [General]
The AUR package `miniplayer-git` was orphaned and so I'm now its maintainer. The AUR install with `yay` (and probably any other helper) does work normally now without assuming dependencies are installed. Apologies in advance if you don't want AUR stuff here or if this is the wrong way of saying this, I didn't this an issue would be correct.

Thanks,
Evan

**Comment by GuardKenzie** [comment]
Thank you so much! 🧡

I'll be in touch regarding updated dependancies when v2 gets close to release.

**Comment by an-prata** [comment]
Small AUR change that might want to be known: `pixcat` doesn't remove its temporary files, so running `miniplayer` for a long time will saturate the resource, crashing `miniplayer` and preventing it from opennig properly for one but also prevent other programs from working correctly (i.e. `rustc` cannot build a project). This is resolved manually by running `rm /tmp/.pixcat-*`, but I also forked `pixcat` and added some cleanup. I made a PR but my hopes of it being merged are low, the author has been innactive for some time, so for the time being the AUR package will use my fork (AUR: `python-pixcat-an-prata`) and save everyone's `tmpfs` from saturation.

To quickly see what I mean just open `miniplayer` and in another terminal run `ls /tmp/.pixcat-* | wc` a few times, the number of files goes up *fast*.

**Reply by an-prata** [reply]
Note that for this change to take effect you may have to remove the user installation of `pixcat`, the fork is version bumped so you can check by seeing if the version is `0.1.5` rather than `0.1.4`. A manual installation of the package through `pip` will likely take precidence over the AUR package.
